help/C/index.docbook

```diff
@@ -2109,30 +2047,6 @@
             </section>
         </section>
 
-        <section xml:id="pluma-indent-lines-plugin">
-            <info>
-                <title>Indent Lines Plugin</title>
-            </info>
-            <para>The <application>Indent Lines</application> plugin adds or removes space from the beginning of lines of text.</para>
-            <para>To indent or unindent text, perform the following steps:</para>
-            <orderedlist inheritnum="ignore" continuation="restarts">
-                <listitem>
-                    <para>Select the lines that you want to indent. To indent or unindent a single line, place the cursor anywhere on that line.</para>
-                </listitem>
-                <listitem>
-                    <itemizedlist>
-                        <listitem>
-                            <para>To indent the text, choose <menuchoice> <guimenu>Edit</guimenu> <guimenuitem>Indent</guimenuitem> </menuchoice>.</para>
-                        </listitem>
-                        <listitem>
-                            <para>To remove the indentation, choose <menuchoice> <guimenu>Edit</guimenu> <guimenuitem>Unindent</guimenuitem> </menuchoice>.</para>
-                        </listitem>
-                    </itemizedlist>
-                </listitem>
-            </orderedlist>
-            <para>The amount of space used, and whether tab character or space characters are used, depends on the <guilabel>Tab Stops</guilabel> settings in the Editor Preferences: see <xref linkend="pluma-prefs-editor"/>.</para>
-        </section>
-
```

Comment:
This edit should be done in a separate commit, since it's not related with the removal of Change Case plugin.

Reply:
Yes, you are right.